README.md

```diff
@@ -18,19 +18,33 @@ Default: diagram.svg
 
 ## `excluded_paths`
 
-A list of paths to exclude from the diagram, separated by commas.
+A list of paths to folders to exclude from the diagram, separated by commas.
```

Comment:
Technically this also worked for paths but only if they were immediate children of `root_dir` (this is why `package.json` worked, but `src/package.json` did not. Rather than explain this nuance, I think it'll be simpler to just only use this config for folders, and nudge people towards mostly using globs instead.